lab04_example/srv/KeyToCmdVel.srv

```diff
@@ -0,0 +1,4 @@
+# Author: Luke Dennis l.dennis@unsw.edu.au
```

Comment:
Add link to where info on services can be found

Reply:
It was given in the lab